.github/workflows/test.yml

```diff
@@ -61,8 +76,12 @@ jobs:
     steps:
       - name: Get code
         uses: actions/checkout@v3
+        with:
+          repository: yash-zededa/eden
```

Comment:
Maybe you should remove your fork from here?

Reply:
I have used fork to test the actions from eden. It works now so we are good to point to lf-edge/eden